app/src/main/java/org/dwoc/khet/gameControl/Piece.kt

```diff
@@ -0,0 +1,31 @@
+package org.dwoc.khet.gameControl
+
+import org.dwoc.khet.models.enums.Direction
+import org.dwoc.khet.models.enums.Player
+
+// Each Types of Piece will Extend this Piece Class
+abstract class Piece(val PlayerId : Player, var PosX : Int, var PosY : Int, var Orientation : Direction)
+{
+
+  //Function MovePiece will set the PosX and PosY of the Piece
```

Comment:
```suggestion
  /**
  * MovePiece will set the PosX and PosY of the Piece
  * @param PosX
  * @param PosY
  */
```
can you change the comments to javadoc style coments